packages/anon-set-db/prisma/schema.prisma

```diff
@@ -0,0 +1,44 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model AnonSet {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  name String
+
+  merkleTrees AnonSetMerkleTree[]
+}
+
+model AnonSetMerkleTree {
+  merkleRoot String @id
+
+  metadata String // NOTE: i.e. versioning info, block num, etc.
+
+  addressesWithMerkleProof AddressWithMerkleProof[]
+  AnonSet                  AnonSet?                 @relation(fields: [anonSetId], references: [id])
+  anonSetId                String?
```

Comment:
@lsankar4033  Do you think we should make this a required foreign key? (remove the "?")